player_android.go

```diff
@@ -279,6 +292,19 @@ func (p *player) Write(data []uint8) (int, error) {
 }
 
 func (p *player) Close() error {
+	p.mu.Lock()
+	p.mu.Unlock()
```

Comment:
> I think Close callers should have responsibility of concurrency,

Ah, understood. Sorry for misunderstanding of whole policy and design.
Let me update again not to use mutexes.